Extensions/AzureAdAuthenticationBuilderExtensions.cs

```diff
@@ -19,28 +20,40 @@ public static AuthenticationBuilder AddAzureAd(this AuthenticationBuilder builde
             return builder;
         }
 
-        private class ConfigureAzureOptions: IConfigureNamedOptions<OpenIdConnectOptions>
+        private class ConfigureAzureOptions : IConfigureNamedOptions<OpenIdConnectOptions>
         {
             private readonly AzureAdOptions _azureOptions;
+            private readonly ITokenAcquisition _tokenAcquisition;
 
-            public ConfigureAzureOptions(IOptions<AzureAdOptions> azureOptions)
+            public ConfigureAzureOptions(IOptions<AzureAdOptions> azureOptions, ITokenAcquisition tokenAcquisition)
             {
                 _azureOptions = azureOptions.Value;
+                _tokenAcquisition = tokenAcquisition;
             }
 
             public void Configure(string name, OpenIdConnectOptions options)
             {
                 options.ClientId = _azureOptions.ClientId;
-                options.Authority = $"{_azureOptions.Instance}common/v2.0";   // V2 specific
+                options.Authority = $"{_azureOptions.Instance}{_azureOptions.TenantId}/v2.0";   // V2 specific
                 options.UseTokenLifetime = true;
                 options.RequireHttpsMetadata = false;
                 options.TokenValidationParameters.ValidateIssuer = false;     // accept several tenants
```

Comment:
I think that @brentschmaltz would want to do better than ValidateIssuer = false.